docs/Tutorials/Advanced-Tutorials/Daily-Quests/Delays.md

```diff
@@ -0,0 +1,116 @@
+---
+icon: material/folder-play-outline
+tags:
+  - Delay
```

Comment:
Yeah I know there are many ways to receive a specific thing but wouldn't it be better for perfomance and for easier understanding/writing to schedules? Correct me if I am wrong